.github/workflows/build_documentation.yml

```diff
@@ -97,7 +97,6 @@ jobs:
           cd doc-build &&
           if [[ `git status --porcelain` ]]; then 
             git add . &&
-            git stash && git pull && git stash apply &&
             git commit -m "Updated with commit ${{ github.sha }} \n\nSee: https://github.com/huggingface/transformers/commit/${{ github.sha }}" &&
             git push origin main
```

Comment:
```suggestion
            git pull
            git push origin main
```

will this work?

Reply:
No, the pull will fail if we don't stash the changes. I will suggest another fix in a few minutes.